samples/net/cloud/google_iot_mqtt/src/main.c

```diff
@@ -59,44 +49,14 @@ void do_sntp(struct addrinfo *addr)
 	} else {
 		LOG_ERR("  Failed to acquire SNTP, code %d\n", rc);
 	}
-
-	sntp_close(&ctx);
+	return rc;
 }
 
 /*
  * TODO: These need to be configurable.
  */
 #define MBEDTLS_NETWORK_TIMEOUT 30000
```

Comment:
This macro does not seem to be used (looks like it never was). Can you remove it (in separate commit), so this sample will get even smaller?

Reply:
done